src/makeCalculator.js

```diff
@@ -37,7 +37,79 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
-}
+  const RESET_DEFAULT_VALUE = 0;
+
+  return {
+    result: 0,
+
+    add(value, result) {
+      return result + value;
+    },
+
+    subtract(value, result) {
+      return result - value;
+    },
+
+    multiply(value, result) {
+      return result * value;
+    },
+
+    divide(value, result) {
+      if (value !== 0) {
+        return result / value;
+      }
+
+      throw Error;
```

Comment:
when you thow an error its good if you add some text about this error (in order to undersatand how to fix it)

in our case you can write: "You cannot divide by zero!"